.travis.yml

```diff
@@ -4,10 +4,10 @@ sudo: required
 dist: bionic
 
 go:
-  - 1.13.x
+  - 1.16.2
 
 env:
-  - GO111MODULE=on
+  - GO111MODULE=off
```

Comment:
Asking out of curiosity, why is this changed to off?

Reply:
The `make test/unit/ci` was kept failing. A solution to it was to either remove the steps from the before_install or change GO111MODULE to off.